<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1071</width>
     <height>841</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton1{
	background-color:rgba(85, 98, 112, 255);
	color:rgba(255, 255, 255, 200);
	border-radius:5px;
}
QPushButton#pushButton1:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(255, 107, 107, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#pushButton1:hover{
	background-color:rgba(255, 107, 107, 255);
}

QPushButton#pushButton2{
	background-color:rgba(85, 98, 112, 255);
	color:rgba(255, 255, 255, 200);
	border-radius:5px;
}
QPushButton#pushButton2:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(255, 107, 107, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#pushButton2:hover{
	background-color:rgba(255, 107, 107, 255);
}

QPushButton#closeButton{
	background-color:rgba(0, 0, 0, 20);
	color:rgba(85, 98, 112, 255);
	border-radius:5px;
}
QPushButton#closeButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(255, 107, 107, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#closeButton:hover{
	background-color:rgba(255, 107, 107, 255);
}</string>
   </property>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>211</y>
      <width>270</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(85, 98, 112, 255), stop:1 rgba(255, 107, 107, 255));
border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>407</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mountain</family>
      <pointsize>150</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 107, 107, 255);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Qatar2022 Arabic</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 255, 255, 220);
font: 75 10pt &quot;Qatar2022 Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;home&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>396</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Qatar2022 Arabic</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Qatar2022 Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>ENCRYPT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>237</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Qatar2022 Arabic Bold</family>
      <pointsize>35</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 255, 255, 220);
font: 75 35pt &quot;Qatar2022 Arabic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>securitext</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Qatar2022 Arabic</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Qatar2022 Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>DECRYPT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>557</x>
      <y>226</y>
      <width>241</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 20);
border-top-right-radius: 15px;
border-bottom-right-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>557</x>
      <y>226</y>
      <width>241</width>
      <height>331</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 255);
border-top-right-radius: 15px;
border-bottom-right-radius: 55px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources/res.qrc">:/Images/ui1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>63</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>232</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Qatar2022 Arabic</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Qatar2022 Arabic&quot;;
border-radius:10px</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <zorder>label_7</zorder>
   <zorder>label_1</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>label_5</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton1</zorder>
   <zorder>pushButton2</zorder>
   <zorder>closeButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
